packages/ui/src/components/JobCard/JobCard.tsx

```diff
@@ -35,49 +39,77 @@ export const JobCard = ({
   jobUrl,
 }: JobCardProps) => {
   const { t } = useTranslation();
+  const JobTitle = <h4 title={`#${job.id}`}>#{job.id}</h4>
+
+  // TODO: Get global config to set the initial state
+  // TODO2: Override to true when its a dedicated job page
+  const [isExpanded, setIsExpanded] = React.useState(true);
```

Comment:
I think that this mode should be an "additional" display mode,
The current state or the collapsable one... it should not be merged to the current one.
Therefore, it is better to extract common components & use a switch for the display mode.

Reply:
I think this can be achieved just modifying this file (and you wont need to maintain twice) and it will only have little impact for users that dont activate the compact mode (basically just having a new arrow button top-right).

I undestand your position and the fear of having such a big diff showing right now in the PR but this really just wrapping everything on a outer container because the card needs to have a header with the id and the button, but the card body is basically the same than right now.

Now:

```
------------------------
| #ID         | Name   |
| TimeLine    | Data   |      
|             |        |
------------------------
```

Then:

```
------------------------
| #ID              Btn |
------------------------
| TimeLine    | Name   |
|             | Data   |      
------------------------
```

But if you are keen to have a different `JobCardCollapsible`, ill extract it.

Reply:
This exraction of the ID look ok 👍